### ISSUE TITLE
Improve deterministic ImageNet baseline 

### ISSUE THREAD

**Issue by dustinvtran** [google/uncertainty-baselines#61]
> @dustinvtran
> I'm thinking of efficientnet or assemble-resnet. Both have TF implementations:
> 
> * https://github.com/clovaai/assembled-cnn
> * https://github.com/tensorflow/tpu/tree/master/models/official/efficientnet/tpu (which I believe is still in the process of converting to Keras/TF 2.0)

https://github.com/google/edward2/issues/207

**Comment by dustinvtran** [comment]
We've had EfficientNet for a few months now. However, there are difficulties still reproducing at least as good the accuracy numbers of the EfficientNet paper due to TF2/Keras/CTL. IIRC, Hieu Pham, TF Model Garden, and an intern are working on this.

**Comment by dustinvtran** [comment]
Duplicate issue: https://github.com/google/edward2/issues/269.
